app/helpers/offender_helper.rb

```diff
@@ -30,4 +30,18 @@ def event_type(event)
       type + 'allocated'
     end
   end
+
+  def approaching_handover_without_com?(offender)
```

Comment:
I'm wondering if this might be a more descriptive name for this method:

```suggestion
  def needs_com_but_ldu_is_uncontactable?(offender)
```

It's a bit longer, but hopefully more descriptive. Not sure if it's too wordy though?

No need to use this, though. Naming things is hard 😓